addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+NEW ir.actions.act_window: account.account_invoice_import_wizard_action
+NEW ir.actions.act_window: account.action_account_payments_transfer
+NEW ir.actions.act_window: account.action_incoterms_tree
+NEW ir.actions.act_window: account.action_open_account_onboarding_invoice_layout
+NEW ir.actions.act_window: account.action_open_account_onboarding_sale_tax
+NEW ir.actions.act_window: account.action_open_account_onboarding_sample_invoice
+NEW ir.actions.act_window: account.action_open_settings (noupdate)
+NEW ir.actions.act_window: account.action_vendor_bill_template
+NEW ir.actions.act_window: account.actions_account_fiscal_year
+NEW ir.actions.act_window: account.invoice_send
+DEL ir.actions.act_window: account.act_account_invoice_partner_relation
+DEL ir.actions.act_window: account.action_account_aged_balance_view
+DEL ir.actions.act_window: account.action_account_balance_menu
+DEL ir.actions.act_window: account.action_account_financial_report_tree
+DEL ir.actions.act_window: account.action_account_general_ledger_menu
+DEL ir.actions.act_window: account.action_account_partner_ledger_menu
+DEL ir.actions.act_window: account.action_account_print_journal_menu
+DEL ir.actions.act_window: account.action_account_report
+DEL ir.actions.act_window: account.action_account_report_bs
+DEL ir.actions.act_window: account.action_account_report_pl
+DEL ir.actions.act_window: account.action_account_tax_report
+DEL ir.actions.act_window: account.action_all_partner_invoices
+DEL ir.actions.act_window: account.action_invoice_tree2
+DEL ir.actions.act_window: account.action_view_account_move_line_reconcile
+DEL ir.actions.act_window: account.action_wizard_multi_chart
+DEL ir.actions.act_window: account.analytic_line_reporting_pivot
+DEL ir.actions.act_window_close: account.setup_wizard_close
+NEW ir.actions.client: account.action_view_account_move_line_reconcile
+DEL ir.actions.report: account.action_report_account_tax
+DEL ir.actions.report: account.action_report_financial
+DEL ir.actions.report: account.action_report_general_ledger
+DEL ir.actions.report: account.action_report_partnerledger
+DEL ir.actions.report: account.action_report_print_overdue
+DEL ir.actions.report: account.action_report_trial_balance
+NEW ir.actions.server: account.action_invoice_tree2
+NEW ir.actions.server: account.model_account_invoice_action_share (noupdate)
+NEW ir.cron: account.ir_cron_reverse_entry
+NEW ir.filters: account.filter_invoice_report_salespersons
+NEW ir.model.access: account.access_account_fiscal_year_manager
+NEW ir.model.access: account.access_account_fiscal_year_user
+NEW ir.model.access: account.access_account_incoterms_all
+NEW ir.model.access: account.access_account_incoterms_manager
+NEW ir.model.access: account.access_account_invoice_line_user
+DEL ir.model.access: account.access_account_financial_report
+DEL ir.model.access: account.access_account_financial_report_invoice
+NEW ir.rule: account.account_payment_term_comp_rule (noupdate)
```

Comment:
We should disable (active=False) this rule, or we will get when migrating missing payment terms in a multi-company environment (as previously there wasn't record rule). It was mentioned in the beginning of the migration issue.